userspace/libsinsp/test/sinsp_with_test_input.h

```diff
@@ -365,7 +365,12 @@ class sinsp_with_test_input : public ::testing::Test {
 
 	// Return true if `field_name` exists in the filtercheck list.
 	// The field value could also be NULL, but in this method, we are not interested in the value.
-	bool field_exists(sinsp_evt *evt, const std::string& field_name, filter_check_list& flist = g_filterlist)
+	bool field_exists(sinsp_evt *evt, const std::string& field_name)
+	{
+		return field_exists(evt, field_name, m_default_filterlist);
+	}
+
+	bool field_exists(sinsp_evt *evt, const std::string& field_name, filter_check_list& flist)
```

Comment:
instead of overloading the methods, wdyt about offering a method to enrich the `m_default_filterlist`? In the plugin test cases, we could just add the plugin filter checks in one shot and not pass them every time we call `field_exists`, `field_has_value`, `get_field_as_string`